### PR TITLE
2018 header update

### PR DESCRIPTION
**Pull request by willpatera**
Update the license header for 2018 and update the license header file to be python 3 compliant. 

Happy 2018!

### CODE REVIEW

File: pupil_src/shared_modules/fixation_detector.py [hunk 1]
```diff
@@ -1,554 +1,564 @@
...
-        self.glfont.set_color_float((0.2, 0.5, 0.9, 1.0))
-
-    def deinit_ui(self):
-        self.remove_menu()
-        self.glfont = None
-
-    def get_init_dict(self):
-        return {'max_dispersion': self.max_dispersion, 'min_duration': self.min_duration,
-                'confidence_threshold': self.confidence_threshold}
+'''
```
Comment (cpicanco) [inline pupil_src/shared_modules/fixation_detector.py]: There is a problem here.

Comment (cpicanco) [reply pupil_src/shared_modules/fixation_detector.py]: Duplicated header.

**Comment by cpicanco** [comment]
Also, there are no license header in some pieces of code, notably `singleeyefitter` and `shared_cpp`.

File: pupil_src/shared_modules/fixation_detector.py (hunk 1 above)
Comment (willpatera) [reply pupil_src/shared_modules/fixation_detector.py]: Fixed, thank you 😄

**Comment by cpicanco** [comment]
Ahhh, `singleeyefitter` has a `licence.md`. What about `shared_cpp`?

**Comment by mkassner** [comment]
@willpatera  any more fixed need for this PR. Can it be merged?

**Pull request merged**